libdnf/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ if(WITH_COMPS)
     target_link_libraries(libdnf ${LIBCOMPS_LIBRARIES})
 endif()
 
+if (WITH_ZCHUNK)
+    pkg_check_modules(ZCHUNKLIB zck>=0.9.11 REQUIRED)
+    add_definitions(-DWITH_ZCHUNK)
+endif()
```

Comment:
Is there a reason for this to be conditionally supported? I think all platforms shipping dnf5 will fully support zchunk since it provides considerable benefit across the board.

Reply:
It is ON by default and for example RHEL 8 does not have zchunk enabled.

Reply:
I'm pretty sure there's a few bugs asking for zchunk support to be added to RHEL 8, but otherwise we could just add it to EPEL.

Reply:
It should be conditional as almost everything else in the code (modularity, comps, ...) especially if it depends on an external lib.